concourse/Makefile

```
@@ -39,6 +39,7 @@ GS                          ?= false
 MINIO                       ?= false
 OEL7                        ?= false
 FILE                        ?= false
+GP6_CLI                     ?= false
 GP7_CLI                     ?= false
```

```suggestion
CLI_GP6                     ?= false
CLI_GP7                     ?= false
```
should we switch such that the name of the feature comes first and the version specifier last ?